RFC-005-SWAP-Basic-Bitcoin.md

```diff
@@ -157,6 +192,39 @@ The funder can use the following witness data to spend the output after the `exp
 
 To be notified of the refund event, both parties MAY watch the blockchain for transactions that spend from the output and check that the witness data is in the above form.
 
+#### Evaluation of the script for refund
+
+**Given** a bitcoin script address that holds the following (hashed) information:
+1. `script code`
+2. `hash of secret`
```

Comment:
Well actually it's the same script for redeem and refund, the wording suggest it's 2 different scripts.